src/components/Header/Header.jsx

```diff
@@ -14,15 +14,15 @@ const Header = () => {
   
     <header className={styles.header}>
       <img src={logo} alt="myteam logo" className={styles.logo} />
-      <nav>
+      <nav className={styles.header-nav}>
         <ul className={ mobileMenu ? '' :  styles['hide-mobile-menu']}>
           <img src={icon_close} alt="" className={styles.icon_close} onClick={toggleMenu} />
-       <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-         
+       <li className={styles.nav-list-item}><Link to="/">Home</Link></li>
+          <li className={styles.nav-list-item}><Link to="/about">About</Link></li>
+           <li className={styles.nav-list-item}><Link to="/contact">Contact Us</Link></li>
```

Comment:
apply the class to the `Link` component to get the colors where you want them:
`<Link to="/contact" className={styles['nav-link']}>` and the other two lines as well

(or with `styles.navLink` if you rename to use camelCase)